entry_types/scrolled/package/spec/contentElements/dataWrapperChart/useIframeHeight-spec.js

```diff
@@ -0,0 +1,19 @@
+import {useIframeHeight} from '../../../src/contentElements/dataWrapperChart/useIframeHeight'
+
+import {renderHook} from '@testing-library/react-hooks';
+import {fakeParentWindow, tick} from 'support';
+import {act} from '@testing-library/react';
+
+describe('useIframeHeight', () => {
+  it('sets the default height', async () => {
+    const testURL = 'https://datawrapper.dwcdn.net/CXXQo/1/';
+    fakeParentWindow();
+
+    const {result} = renderHook(() => useIframeHeight(testURL));
+
+    window.postMessage('SOME_MESSAGE', '*');
+    await tick();
+       
+    expect(result.current).toEqual('400px');
+  });
```

Comment:
Consider adding a spec that tests whether we can successfully remove the event listener. This has the advantage that in case somebody breaks this functionality, there will be a failing spec and we'd notice before the merge. For an example, see [1]

[1] https://github.com/codevise/pageflow/blob/f4f4f9523df802c82990bb2ce96fcee5936462fb/entry_types/scrolled/package/spec/frontend/usePostMessageListener-spec.js#L18